### ISSUE TITLE
Difference between console.log and console.warn

### ISSUE THREAD

**Issue by nishantwrp** [gridsome/gridsome#1200]
## Summary

I was developing a plugin for gridsome and wanted to give a warning using `console.warn` but I noticed that the warning looks pretty similar to `console.log` it should be different than that.

### Basic example
We could differentiate them using colors?

**Comment by noxify** [comment]
Not sure if this is related to gridsome 🙈

Anyway... I solved it by using `chalk`.

Here an example:

```

console.log(`${chalk.yellowBright('Remote images are not downloaded. Please check your configuration.')}`)

```

**Comment by nishantwrp** [comment]
Hmm, gatsby does this, by adding a yellow-colored label behind the message `warn`.

**Comment by noxify** [comment]
Ok - but do we need this in the core? 
I mean, a colored console log is, for me, more a nice to have.

Is it not a solution for you to you chalk if you need a colored console output? 

Please don’t understand me wrong :) 
If I have missed something, please feel free to correct me.

**Comment by nishantwrp** [comment]
IMO, using `chalk` whenever a warning needs to be thrown doesn't seem correct.

**Comment by hjvedvik** [comment]
I agree with @noxify that we shouldn't override `console.warn`. Gridsome has some simple log utils [here](https://github.com/gridsome/gridsome/blob/master/gridsome/lib/utils/log.js), which are used internally for colored messages.  But I think they need some improvements if they are going to be exposed to plugins.

**Comment by nishantwrp** [comment]
Oh, ok. So yeah, maybe the scope of this issue should be expose those to plugins.
